### PR TITLE
Automates release pipeline for varats/varats-core

### PR DESCRIPTION
**Pull request by MiriamMarx**

### CODE REVIEW

File: .github/workflows/tag.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Python Release Tag Pipeline
+
+on:
+  workflow_dispatch: # TODO: for testing only
```
Comment (MiriamMarx) [inline .github/workflows/tag.yml]: drop line after testing

File: .github/workflows/tag.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - uses: ./.github/actions/dependenciesActions
+    - name: Create and Publish Release
+      run: |
+        sed -i 's/$PYPI_TOKEN/'$PYPI_TOKEN'/' .pypirc
+        sed -i 's/$TEST_PYPI_TOKEN/'$TEST_PYPI_TOKEN'/' .pypirc
+        cp .pypirc $HOME # to ensure tox uses the correct pypi config
+        tox -e test-release # TODO: remove test- for production
```
Comment (MiriamMarx) [inline .github/workflows/tag.yml]: fix TODO after testing

File: .github/workflows/tag.yml (hunk 1 above)
Comment (vulder) [inline .github/workflows/tag.yml]:
```suggestion
```

**Review by vulder — APPROVED** [review]
Looks good, let's try this.

Thx, for contributing

**Pull request merged**